packages/cedar-amcharts/src/render/render.ts

```diff
@@ -32,8 +32,23 @@ export function renderChart(elementId: string, definition: any, data?: any) {
     spec = merge(spec, definition.overrides, { clone: true })
   }
 
-  const chart = AmCharts.makeChart(elementId, spec)
-  return
+  const chart: any = AmCharts.makeChart(elementId, spec)
+  // If it is a pie chart
+  if (definition.type === 'pie') {
+    // Set pie chart balloonText
+    chart.balloonText = getPieBalloonText(definition)
+  }
+
+  return chart
```

Comment:
We also need to return the chart above in the `if (definition.type === 'custom') {` block. Probably best to `let chart: any` at the top.